CONTRIBUTING.md

```diff
@@ -24,10 +28,50 @@
     - only applicable to snippet import
 
 ### Setting Up Development
+
 - Cloning, installing, and building
 ```
 git clone git@github.com:amplitude/Amplitude-JavaScript.git
 cd Amplitude-JavaScript
 make # Runs tests and generate builds
 yarn dev # Start development utility. Open localhost:9000 in your browser to access
 ```
+
+### PR Commit Title Conventions
+
+PR titles should follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/). A [probot app](https://github.com/zeke/semantic-pull-requests) checks for this when a PR is opened. This helps automate the [release](#release) process.
+
+#### Commit Types ([related to release conditions](#release))
+
+- **Special Case**: Any commit with `BREAKING CHANGES` in the body: Creates major release
+- `feat(<optional scope>)`: New features (minimum minor release)
+- `fix(<optional scope>)`: Bug fixes (minimum patch release)
+- `perf(<optional scope>)`: Performance improvement
+- `docs(<optional scope>)`: Documentation updates
+- `test(<optional scope>)`: Test updates
+- `refactor(<optional scope>)`: Code change that neither fixes a bug nor adds a feature
+- `style(<optional scope>)`: Code style changes (e.g. formatting, commas, semi-colons)
+- `build(<optional scope>)`: Changes that affect the build system or external dependencies (e.g. Yarn, Npm)
+- `ci(<optional scope>)`: Changes to our CI configuration files and scripts
+- `chore(<optional scope>)`: Other changes that don't modify src or test files
+- `revert(<optional scope>)`: Revert commit
+
+### Release
```

Comment:
You can say `Release [Amplitude Internal]`, since external contributors won't care.

Reply:
Updated